transform/mattermost-analytics/models/staging/hacktoberboard_prod/_hacktoberboard_prod__models.yml

```diff
@@ -22,3 +22,83 @@ models:
         description: The ID of the server the event originated from.
       - name: received_at
         description: Timestamp registered by RudderStack when the event was ingested (received).
+      - name: timestamp
+        description: Timestamp registered by RudderStack when the event occurred refactored to clients clock skew.
+
+
+  - name: stg_hacktoberboard_prod__activity
```

Comment:
Even Chen's query he shared in the channel, hes directly hitting the `FOCALBOARD_EVENT_TELEMETRY` table, and calculating the metrics hes looking at, IMO this source isn't required and can be dropped, as it might create inconsistencies (if they exist).

Reply:
I did compare them. There's significant difference. Seems that front-end telemetry doesn't report the full user actions, thus greatly under-reporting. A quick query on the fact table for comparing the sum of front-end vs server activity will show the difference.

Reply:
Yes, thats what I thought, there was a known issue in boards, when a user triggers or fires an action, the `rudderTrack()` was fired multiple times and this issue was fixed by the Dev team.

Reply:
@sri-byte this shouldn't affect active users, as multiple events are aggregated per day.